### PR TITLE
Add support for Adafruit Feather RP2040 with RFM95.

### PR DESCRIPTION
**Pull request by miltieIV2**
## Thank you for sending in a pull request, here's some tips to get started!

(Please delete all these tips and replace with your text)

- Before starting on some new big chunk of code, it it is optional but highly recommended to open an issue first
  to say "hey, I think this idea X should be implemented and I'm starting work on it. My general plan is Y, any feedback
  is appreciated." This will allow other devs to potentially save you time by not accidentially duplicating work etc...
- Please do not check in files that don't have real changes
- Please do not reformat lines that you didn't have to change the code on
- We recommend using the [Visual Studio Code](https://platformio.org/install/ide?install=vscode) editor along with the ['Trunk Check' extension](https://marketplace.visualstudio.com/items?itemName=trunk.io) (In beta for windows, WSL2 for the linux version),
  because it automatically follows our indentation rules and its auto reformatting will not cause spurious changes to lines.
- If your PR fixes a bug, mention "fixes #bugnum" somewhere in your pull request description.
- If your other co-developers have comments on your PR please tweak as needed.
- Please also enable "Allow edits by maintainers".

### CODE REVIEW

**Comment by miltieIV2** [comment]
See https://github.com/meshtastic/firmware/discussions/4405

**Comment by miltieIV2** [comment]
Move meshtastic_HardwareModel enum 72 -> 75

**Comment by miltieIV2** [comment]
CLA signed

File: src/mesh/generated/meshtastic/mesh.pb.h [hunk 1]
```diff
@@ -187,6 +187,8 @@ typedef enum _meshtastic_HardwareModel {
     /* RadioMaster 900 Bandit, https://www.radiomasterrc.com/products/bandit-expresslrs-rf-module
  SSD1306 OLED and No GPS */
     meshtastic_HardwareModel_RADIOMASTER_900_BANDIT = 74,
+    /* AdaFruit Feather RP2040 RFM95 with optional SSD1306 display */
+    meshtastic_HardwareModel_RP2040_FEATHER_RFM95 = 75,
```
Comment (GUVWAF) [inline src/mesh/generated/meshtastic/mesh.pb.h:191]: These files are automatically generated. Changes would need to be added to the https://github.com/meshtastic/protobufs repository.

Comment (miltieIV2) [reply src/mesh/generated/meshtastic/mesh.pb.h:191]: Ah, I need to do Step 3 here:
https://meshtastic.org/docs/development/firmware/build/#distribute--publish-custom-builds

On the way...

**Comment by miltieIV2** [comment]
Comment added to protobuf PR: https://github.com/meshtastic/protobufs/pull/559

```
This is the first platform.io Adafruit Feather RP2040 board.
There is a of course a lot of pin overlap with other RP2040, but this is the first USE_RF95 RP2040.
```

**Pull request merged**